### DISCUSSION TITLE
Welcome to goaccess Discussions!

### DISCUSSION THREAD

**Discussion by allinurl** [allinurl/goaccess#2193] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

If you have a bug to report or a feature to suggest, please open an issue, so that the maintainers can react to it and track it's status.
If you have a question, a concern or an idea, please open a discussion, or search for an existing one, here, or in one of these:

**Comment by sureshvv** [comment]
Thank you for creating this.
